src/lib.rs

```diff
@@ -10,6 +10,8 @@
 #![warn(missing_docs)]
 #![deny(missing_debug_implementations)]
 #![deny(unsafe_op_in_unsafe_fn)]
+// Some of our non-instructions docs link to instructions-only functionality.
+#![cfg_attr(not(feature = "instructions"), allow(rustdoc::broken_intra_doc_links))]
```

Comment:
This attribute can be added directly to the `GlobalDescriptorTable::from_raw_entries`.

Reply:
Done